src/main/java/duke/ParsedAnswerHandler.java

```diff
@@ -0,0 +1,118 @@
+package duke;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * This class receives a ParsedAnswer object and execute commands based on information
+ * from the object.
+ */
+
+public class ParsedAnswerHandler {
+    ParsedAnswer pa;
+
+    ParsedAnswerHandler(ParsedAnswer pa) {
+        this.pa = pa;
+    }
+
+    void execute() {
+        String filePath = System.getProperty("user.dir")  + "/data/storage.txt";
+        switch (pa.getCommand()) {
+            case "bye":
```

Comment:
According to the coding standard, the "case" should be aligned with "switch" without an indentation.